Up81

Ryan-Guinn opened the pull request.

Ryan-Guinn committed:
Added routing for Plan page

Ryan-Guinn committed:
created main plan page

Ryan-Guinn committed:
Added all of the files I anticipate needing

Ryan-Guinn committed:
completed Plan By ID query

Ryan-Guinn committed:
add query for getting all plans

Ryan-Guinn committed:
fixed the file that I editted

Ryan-Guinn committed:
Finalized the planInterface

Ryan-Guinn committed:
added all of the fields needed on the UI

Ryan-Guinn committed:
removed extraneous files

fontainesw commented on packages/ui-bindings/Plan/planInterface.tsx:
```diff
@@ -0,0 +1,57 @@
+  workingAgents: [Agent]
+  plannedNonWorkInputs: any // TODO: replace with [Commitment]
+  plannedOutputs: any // TODO: replace with [Commitment]
+  nonWorkInputs: any // TODO: replace with [EconomicEvent]
```
Economic event already exists in the project, but It dosen't have a type defined. you should define it in the interface file for economic event, and then use that here

Ryan-Guinn replied:
added

fontainesw commented on packages/app/routes.tsx:
```diff
@@ -65,6 +66,7 @@ const routes = (
+      <Route path="plan" component={Plan}/>
       <Route path="person" component={PersonPage} />
     </Route>
   </Route>
```
The "export default routes" needs a semicolon

Ryan-Guinn replied:
fixed

fontainesw reviewed (commented):
Looks pretty good, I just have 2 things

Ryan-Guinn committed:
Added a semicolon to the routes.tsx file as per request of Steven on the Pull Request

Ryan-Guinn committed:
Modified Plan Interface to properly import econimcEvent
I'm not entirely sure why there was already an import in place but it doesn't seem to affect anything.

Ryan-Guinn committed:
Created an interface for EconomicEvent

ConnorHibbs commented on packages/ui-bindings/EconomicEvent/economicEventInterface.tsx:
```diff
@@ -0,0 +1,64 @@
+/**
```
Is this a new file? There is already a file with this same name and path in Master that defined the GraphQL fragment for EconomicEvent.

ConnorHibbs reviewed (approved):
Looks pretty good. I had a question about EconomicEvent.tsx but code wise everything is good.
